examples/activities/resample.js

```diff
@@ -62,19 +69,28 @@ module.exports = function resample(localInputPath, outputUri, options, callback)
         return done.apply(null, arguments);
       }
 
-      var ds = gdal.open(localOutputPath);
-
-      try {
-        ds.bands.get(1).getStatistics(false, true);
-      } catch (err) {
-        // likely "Failed to compute statistics, no valid pixels found in
-        // sampling.", which means that it's all NODATA, and thus not worth
-        // saving
-
-        return done();
-      } finally {
-        ds.close();
-      }
+      // try {
```

Comment:
This turned out to be really slow, and wasn't worth the couple of empty border tiles it was removing from the set in `oam-server`. Since we're only tiling to the image's native resolution, the ND tile concerns are lessened. Perhaps there's another way to remove ND tiles, but performance really increased for my local tiler when I commented all this out.